ecchronos-binary/src/test/bin/pytests.sh

```diff
@@ -68,6 +68,27 @@ sed 's;^\(\s*\)\(<appender-ref ref="STDOUT" />\)\s*$;\1<!-- \2 -->;g' -i "$CONF_
 sed "s/#\?scheduler.run.interval.time=[0-9]\+/scheduler.run.interval.time=1/g" -i "$CONF_DIR"/ecc.cfg
 sed "s/#\?scheduler.run.interval.time.unit=seconds\+/scheduler.run.interval.time.unit=seconds/g" -i "$CONF_DIR"/ecc.cfg
 
+## Special config for test.table1
+
+cat <<EOF > "$CONF_DIR"/schedule.yml
```

Comment:
Nit: should we test the wildcard here too?

Reply:
Good point, I was considering doing that but decided against it. IMO the main part is making sure we pick up the configuration and apply it, the rest should be covered by unit tests.